brave/src/test/java/com/linecorp/armeria/common/brave/RequestContextCurrentTraceContextTest.java

```diff
@@ -17,71 +17,50 @@
 package com.linecorp.armeria.common.brave;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.ArgumentMatchers.isA;
 import static org.mockito.Mockito.when;
 
 import java.util.Collections;
 
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.mockito.Answers;
+import javax.annotation.Nullable;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnit;
-import org.mockito.junit.MockitoRule;
-import org.mockito.stubbing.Answer;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
 
+import com.linecorp.armeria.common.HttpMethod;
+import com.linecorp.armeria.common.HttpRequest;
 import com.linecorp.armeria.common.RequestContext;
 import com.linecorp.armeria.common.util.SafeCloseable;
-import com.linecorp.armeria.internal.DefaultAttributeMap;
 import com.linecorp.armeria.internal.brave.TraceContextUtil;
 import com.linecorp.armeria.internal.brave.TraceContextUtil.PingPongExtra;
+import com.linecorp.armeria.server.ServiceRequestContext;
 
 import brave.propagation.CurrentTraceContext;
 import brave.propagation.CurrentTraceContext.Scope;
 import brave.propagation.TraceContext;
 import io.netty.channel.EventLoop;
-import io.netty.util.Attribute;
-import io.netty.util.AttributeKey;
 
-public class RequestContextCurrentTraceContextTest {
-    @Rule
-    public MockitoRule mockitoRule = MockitoJUnit.rule();
+@ExtendWith(MockitoExtension.class)
```

Comment:
Shouldn't need this, `testing-internal` enables this extension automatically

Reply:
Oops I didn't notice that. Thanks!